localstack/config.py

```diff
@@ -453,6 +453,9 @@ def is_linux():
 # Equivalent to HTTPS_PROXY, but only applicable for external connections
 OUTBOUND_HTTPS_PROXY = os.environ.get("OUTBOUND_HTTPS_PROXY", "")
 
+# Whether to enable the partition adjustment listener (in order to support other partitions that the default)
+PARTITION_ADJUSTMENT = is_env_true("PARTITION_ADJUSTMENT")
```

Comment:
i'm not sure about htis naming here. both the word partition and adjustment are very ambiguous and it's not clear what it does. i would maybe go for something less generic and more concrete like `GOV_CLOUD_SUPPORT`

Reply:
IMO `GOV_CLOUD_SUPPORT` would be bit too specific since this listener also works with other partitions such as `cn` or the `iso` partitions.

Reply:
Then maybe we should call it `ARN_PARTITION_REWRITING`?

Reply:
Renamed to `ARN_PARTITION_REWRITING`.